tests/acceptance/logout-test.js

```diff
@@ -0,0 +1,28 @@
+import Ember from "ember";
+import { module, test } from 'qunit';
```

Comment:
'import' is only available in ES6 (use esnext option).<br>Mixed double and single quotes.